python/ray/serve/llm/examples/pd_disagg/app.py

```diff
@@ -0,0 +1,205 @@
+"""Using Ray Serve to deploy LLM models with P/D disaggregation.
+"""
+import asyncio
+import logging
+from typing import AsyncGenerator, Union
+
+from pydantic import BaseModel
+from vllm.config import KVTransferConfig
+
+from ray import serve
+from ray.serve.deployment import Application
+from ray.serve.handle import DeploymentHandle
+from ray.serve.llm import (
+    LLMRouter,
+    LLMConfig,
+    ModelLoadingConfig,
+    build_llm_deployment,
+    LLMServer,
+)
+
+from ray.llm._internal.serve.configs.prompt_formats import Prompt
+from ray.llm._internal.serve.configs.server_models import (
+    parse_args as parse_llm_configs,
+    LLMRawResponse,
+)
+from ray.llm._internal.serve.deployments.llm.llm_server import ResponsePostprocessor
```

Comment:
ok there are so many private APIs here that I don't think it makes sense to expose to the users. I am coming to conclusion that we should refactor this PR in a follow up to not put this as an example. An example that uses `_internal` api is not a great.